openslides_backend/action/speaker/create_update_delete.py

```diff
@@ -14,6 +19,28 @@ class SpeakerCreateAction(CreateActionWithInferredMeeting):
         optional_properties=["marked"],
     )
 
+    def validate_fields(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Checks that a new speaker does not already exist on the list of speaker as
+        comming speaker (with begin_time == None)
+        """
+        filter_obj = And(
+            FilterOperator("list_of_speakers_id", "=", instance["list_of_speakers_id"]),
+            FilterOperator("end_time", "=", None),
```

Comment:
Can you add a speaker another time if he is currently speaking? If yes, the check for `begin_time == None` is missing here. If not, the comment above is false and should say `end_time`.

Reply:
Fixed.